src/main/java/org/gridsuite/modification/server/dto/ModificationInfos.java

```diff
@@ -35,33 +35,34 @@
-    @JsonSubTypes.Type(value = EquipmentDeletionInfos.class, name = "EQUIPMENT_DELETION"),
-    @JsonSubTypes.Type(value = LineSplitWithVoltageLevelInfos.class, name = "LINE_SPLIT_WITH_VOLTAGE_LEVEL"),
-    @JsonSubTypes.Type(value = LineAttachToVoltageLevelInfos.class, name = "LINE_ATTACH_TO_VOLTAGE_LEVEL"),
-    @JsonSubTypes.Type(value = LinesAttachToSplitLinesInfos.class, name = "LINES_ATTACH_TO_SPLIT_LINES"),
-    @JsonSubTypes.Type(value = BranchStatusModificationInfos.class, name = "BRANCH_STATUS_MODIFICATION"),
-    @JsonSubTypes.Type(value = EquipmentAttributeModificationInfos.class, name = "EQUIPMENT_ATTRIBUTE_MODIFICATION"),
-    @JsonSubTypes.Type(value = GeneratorScalingInfos.class, name = "GENERATOR_SCALING"),
-    @JsonSubTypes.Type(value = LoadScalingInfos.class, name = "LOAD_SCALING"),
-    @JsonSubTypes.Type(value = DeleteVoltageLevelOnLineInfos.class, name = "DELETE_VOLTAGE_LEVEL_ON_LINE"),
-    @JsonSubTypes.Type(value = DeleteAttachingLineInfos.class, name = "DELETE_ATTACHING_LINE"),
-    @JsonSubTypes.Type(value = GenerationDispatchInfos.class, name = "GENERATION_DISPATCH")
+    @JsonSubTypes.Type(value = GroovyScriptInfos.class),
+    @JsonSubTypes.Type(value = BatteryCreationInfos.class),
+    @JsonSubTypes.Type(value = LoadCreationInfos.class),
+    @JsonSubTypes.Type(value = LoadModificationInfos.class),
+    @JsonSubTypes.Type(value = GeneratorCreationInfos.class),
+    @JsonSubTypes.Type(value = GeneratorModificationInfos.class),
+    @JsonSubTypes.Type(value = LineCreationInfos.class),
+    @JsonSubTypes.Type(value = LineModificationInfos.class),
+    @JsonSubTypes.Type(value = SubstationCreationInfos.class),
+    @JsonSubTypes.Type(value = SubstationModificationInfos.class),
+    @JsonSubTypes.Type(value = VoltageLevelCreationInfos.class),
+    @JsonSubTypes.Type(value = VoltageLevelModificationInfos.class),
+    @JsonSubTypes.Type(value = ShuntCompensatorCreationInfos.class),
+    @JsonSubTypes.Type(value = ShuntCompensatorModificationInfos.class),
+    @JsonSubTypes.Type(value = TwoWindingsTransformerCreationInfos.class),
+    @JsonSubTypes.Type(value = TwoWindingsTransformerModificationInfos.class),
+    @JsonSubTypes.Type(value = EquipmentDeletionInfos.class),
+    @JsonSubTypes.Type(value = LineSplitWithVoltageLevelInfos.class),
+    @JsonSubTypes.Type(value = LineAttachToVoltageLevelInfos.class),
+    @JsonSubTypes.Type(value = LinesAttachToSplitLinesInfos.class),
+    @JsonSubTypes.Type(value = BranchStatusModificationInfos.class),
+    @JsonSubTypes.Type(value = EquipmentAttributeModificationInfos.class),
+    @JsonSubTypes.Type(value = GeneratorScalingInfos.class),
+    @JsonSubTypes.Type(value = LoadScalingInfos.class),
+    @JsonSubTypes.Type(value = DeleteVoltageLevelOnLineInfos.class),
+    @JsonSubTypes.Type(value = DeleteAttachingLineInfos.class),
+    @JsonSubTypes.Type(value = GenerationDispatchInfos.class),
+    @JsonSubTypes.Type(value = MassiveEquipmentsModificationsInfos.class)
```

Comment:
why this change ?

Reply:
The 'name' properties are no longer necessary here since we added a JsonTypeName annotation on each modification in previous PR